var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -198,27 +225,36 @@ class Cp2k(MakefilePackage, CudaPackage):
         depends_on("sirius@7.0.0:7.0", when="@8:8.2")
         depends_on("sirius@7.2", when="@8.3:8.9")
         depends_on("sirius@7.3:", when="@9.1")
-        conflicts("~mpi")
+        depends_on("sirius@7.4:", when="@master")
+        conflicts("~mpi", msg="SIRIUS requires MPI")
         # sirius support was introduced in 7+
         conflicts("@:6")
 
     with when("+libvori"):
-        depends_on("libvori@201219:", when="@8.1", type="build")
-        depends_on("libvori@210412:", when="@8.2:", type="build")
+        depends_on("libvori@201219:", when="@8.1")
+        depends_on("libvori@210412:", when="@8.2:")
+        depends_on("libvori@220621:", when="@2023.1:")
         # libvori support was introduced in 8+
         conflicts("@:7")
 
     # the bundled libcusmm uses numpy in the parameter prediction (v7+)
     # which is written using Python 3
-    depends_on("py-numpy", when="@7:+cuda", type="build")
-    depends_on("python@3.6:", when="@7:+cuda", type="build")
+    depends_on("py-numpy", when="@7:+cuda")
+    depends_on("python@3.6:", when="@7:+cuda")
+    depends_on("py-fypp")
```

Comment:
I just noticed that this dependency causes `cp2k` to depend on `python`. Is that really necessary?
Is it just a build dependency?

Reply:
`py-fypp` should be a build dependency AFAIK.